eZ/Bundle/EzPublishCoreBundle/Command/ReindexCommand.php

```diff
@@ -374,12 +383,13 @@ private function getStatementContentSince(DateTime $since, $count = false)
     /**
      * @param mixed $locationId
      * @param bool $count
+     * @param int $offset
      *
      * @return \Doctrine\DBAL\Driver\Statement
      *
      * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException
      */
-    private function getStatementSubtree($locationId, $count = false)
+    private function getStatementSubtree($locationId, $count = false, int $offset = 0)
```

Comment:
`$offset` parameter is always passed so there is no need for default value. 

_It's private method so you can safely change the method signature without BC break._